### ISSUE TITLE
License should reflect OSL

### ISSUE THREAD

**Issue by fooman** [magento/magento2#17225]
### Preconditions

1. Check license stated in composer.json for packages
2. Compare to license.txt

### Steps to reproduce

1. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/AdvancedSearch/composer.json#L19
2. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Amqp/composer.json#L14
3. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/AsynchronousOperations/composer.json#L21
4. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Authorizenet/composer.json#L22
5. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Braintree/composer.json#L31
6. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Elasticsearch/composer.json#L21
7. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/EncryptionKey/composer.json#L14
8. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/MessageQueue/composer.json#L13
9. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/MysqlMq/composer.json#L14
10. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Paypal/composer.json#L32
11. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/ProductVideo/composer.json#L22
12. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Signifyd/composer.json#L23
13. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/SwaggerWebapiAsync/composer.json#L16
14. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Swatches/composer.json#L25
15. https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Vault/composer.json#L18

The following Core Bundled Extension
vertex/module-tax
is currently stated as proprietary but no further license details are provided.

### Expected result

1. license in composer.json should read 
```
    "license": [
        "OSL-3.0",
        "AFL-3.0"
    ],
```

### Actual result

1. proprietary

**Comment by magento-engcom-team** [comment]
Hi @fooman. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@fooman do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by engcom-backlog-nickolas** [comment]
Hello @fooman, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by sidolov** [comment]
Hi @fooman. Thank you for your report. 
The issue has been fixed in magento/magento2#17268 by @mhauri in 2.3-develop branch
Related commit(s):
 - [90134aef95bdd3689f8236eab233b09e6c84a9a1](https://github.com/magento/magento2/commit/90134aef95bdd3689f8236eab233b09e6c84a9a1)
 - [0d5cb1e4d2f74eab03c637c68cb820e5adab75e0](https://github.com/magento/magento2/commit/0d5cb1e4d2f74eab03c637c68cb820e5adab75e0)

The fix will be available with the upcoming 2.3.0 release.

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by TomashKhamlai** [comment]
The issue was re-tested and we can confirm that it was fixed on the 2.3 release branch. We closing this issue as fixed due to upcoming 2.3 release that will be available soon.

**Comment by cscd98** [comment]
@TomashKhamlai Define soon please.
